### PR TITLE
added variables

### PR DESCRIPTION
**Pull request by jasminedns**
created variables css file inside styles and added variables

### CODE REVIEW

**Review by idacarlin95 — APPROVED** [review]
lgtm

**Comment by potnurusrilatha** [comment]
looks good

**Comment by potnurusrilatha** [comment]
checked it looks good

**Review by potnurusrilatha — APPROVED** [review]
looks good

File: styles/variables.css [hunk 1]
```diff
@@ -0,0 +1,16 @@
+:root {
+    --small-font: 19px
+    --base-font: 22px
+    --large-font: 64px
+    --textcolor-main: #211000;
+    --textcolor-secondary: #F5F0D7;
+    --title-main: #3C5724;
+    --title-secondary: #89AA4F;
+    --title-terceiary: #F5F0D7;
```
Comment (TobiasKjernell) [inline styles/variables.css]: tertiary (miss spelling)

**Pull request merged**